client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -3,66 +3,148 @@
 /**
  * External dependencies
  */
-import classNames from 'classnames';
 import { BlockControls, InspectorControls } from '@wordpress/editor';
-import {
-	Button,
-	PanelBody,
-	RangeControl,
-	ToggleControl,
-	Toolbar,
-	Path,
-	SVG,
-} from '@wordpress/components';
+import { PanelBody, RangeControl, ToggleControl, Toolbar, Path, SVG } from '@wordpress/components';
 import { Component, Fragment } from '@wordpress/element';
 import { get } from 'lodash';
 import { withSelect } from '@wordpress/data';
+import { compose, withInstanceId } from '@wordpress/compose';
 
 /**
  * Internal dependencies
  */
 import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 
-export const MAX_POSTS_TO_SHOW = 3;
+export const MAX_POSTS_TO_SHOW = 6;
 
 function PlaceholderPostEdit( props ) {
-	const previewClassName = 'related-posts__preview';
-
 	return (
-		<div className={ `${ previewClassName }-post` }>
+		<div
+			className="jp-related-posts-i2__post"
+			id={ props.id }
+			aria-labelledby={ props.id + '-heading' }
+		>
+			<strong id={ props.id + '-heading' } class="jp-related-posts-i2__post-link">
```

Comment:
You would most likely need to use `className` here, to avoid React warnings:

```suggestion
			<strong id={ props.id + '-heading' } className="jp-related-posts-i2__post-link">
```